src/services/databaseservice.cpp

```diff
@@ -876,6 +876,11 @@ bool DatabaseService::setupTables() {
         version = 41;
     }
 
+    if (version < 42) {
+        queryDisk.exec(QStringLiteral("ALTER TABLE calendarItem ADD tags VARCHAR(255)"));
```

Comment:
I hope no one has more tags than fitting in 255 char 😉

Reply:
Heh, yeah, I think I'll set that higher value.

Reply:
I set it to 9999, but if there is better setting for this let me know! :)

Reply:
SQLite doesn't seem to enforce that limit anyway... I'd just use 512 or so...

Reply:
Ah, okay, I'll set it to 512 then :D